core/src/test/java/jenkins/util/io/PathRemoverTest.java

```diff
@@ -418,6 +424,34 @@ public void testForceRemoveRecursive_ParentIsSymbolicLink() throws IOException {
         assertFalse(d1.exists());
     }
 
+    @Test
+    @Issue("JENKINS-55448")
+    public void testForceRemoveRecursive_TruncatesNumberOfExceptions() throws IOException {
+        assumeTrue(Functions.isWindows());
```

Comment:
Not sure how to test this on other platforms without a ton of mocking, which I wanted to avoid.